packages/evm/test/disputes/disputes-create.js

```diff
@@ -175,14 +174,15 @@ contract('DisputeManager', () => {
     })
 
     context('when the sender is not an arbitrable', () => {
-      let fakeArbitrable
+      it('creates a dispute', async () => {
+        const { disputeFees } = await courtHelper.getDisputeFees()
+        await courtHelper.mintFeeTokens(fakeArbitrable, disputeFees)
+        await feeToken.approve(disputeManager.address, disputeFees, { from: fakeArbitrable })
 
-      beforeEach('mock non arbitrable', async () => {
-        fakeArbitrable = await FakeArbitrable.new(court.address)
-      })
+        const receipt = await court.createDispute(2, '0xabcd', { from: fakeArbitrable })
 
-      it('reverts', async () => {
-        await assertRevert(fakeArbitrable.createDispute(2, '0x'), ARAGON_COURT_ERRORS.SENDER_NOT_ARBITRABLE)
+        assertAmountOfEvents(receipt, DISPUTE_MANAGER_EVENTS.NEW_DISPUTE, { decodeForAbi: DisputeManager.abi })
+        assertEvent(receipt, DISPUTE_MANAGER_EVENTS.NEW_DISPUTE, { expectedArgs: { disputeId: 0, subject: fakeArbitrable, metadata: '0xabcd' }, decodeForAbi: DisputeManager.abi })
       })
     })
```

Comment:
Should we add a test here to call the `arbitrator.rule()` function from the EOA account?

Reply:
I think we were already testing it [here](https://github.com/aragon/protocol/blob/development/packages/evm/test/disputes/disputes-settle-round.js#L72)